modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java

```diff
@@ -1079,6 +1169,12 @@ private void copyDefaultStructs() {
         copyStruct("IllegalArgumentException", "Exception", "Object");
         copyStruct("IllegalStateException", "Exception", "Object");
         copyStruct("NumberFormatException", "Exception", "Object");
+        
+        copyStruct("GeoPoint", "Object");
+        copyStruct("Strings", "List<String>", "Collection<String>", "Object");
+        copyStruct("Longs", "List<Object>", "Collection<Object>", "Object");
+        copyStruct("Doubles", "List<Object>", "Collection<Object>", "Object");
+        copyStruct("GeoPoints", "List<Object>", "Collection<Object>", "Object");
```

Comment:
I think for Longs, Doubles, and GeoPoints it's safer to use List (List<def>) so as an example Longs would be

copyStruct("Longs", "List", "Collection", "Object");

The reason being is that if anyone ever casts to one of these then we probably want to be able to handle the Object coming out of it without requiring a second cast.

((Longs)longField).get(0) is now a def type instead of an Object type.  Really rare that this would ever happen, but I think for consistency this could be important.